patches/minecraft/net/minecraft/entity/passive/AbstractHorse.java.patch

```diff
@@ -17,37 +9,34 @@
  
          if (!this.field_70170_p.field_72995_K && this.field_110296_bG != null)
          {
-+        	captureDrops = true;
++            captureDrops = true;
 +        	this.capturedDrops.clear();
```

Comment:
This is also a tab. I suggest you to use Ctrl+F to find tabs and replace them with 4 spaces.